README.md

```diff
@@ -1,2 +1,6 @@
-# plp-samples
+# plp-demo-app
```

Comment:
please merge the 2gp README content into this README, so it's at the top level of repo. due to the multiple link-outs from the doc, all files are expected to be from home dir and not nested under /2gp-demo/

we can differentiate between 1gp and 2gp with two sections in the README, esp if there's no material difference in the code/metadata itself. it's harder to maintain two code bases for every future change

Reply:
@minhajkhansfdc - As discussed in our voice chat on Friday, putting 1GP and 2GP instructions in a single README is not recommended.

I will create two READMEs, one for each packaging method, and will keep them both at the root level. The "main" README will offer some setup instructions (e.g. "make sure you have the CLI installed and clone this repo"), and will have links to the 1GP or 2GP instructions.